solutions/platinum/cf-600E.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: cf-600E
+source: CF
+title: Lomsat gelral
+author: Chongtian Ma
+---
+
+[Official Analysis](https://codeforces.com/blog/entry/21827)
+
+## Implementation
+
+As explained in the official analysis, when we use small to large merging on data structures such as sets and maps, we optimize the amount of operations from $$\mathcal{O}(N^2 \log N)$ to $\mathcal{O}(N \log^2 N)$. We can take advantage of this to store all kinds of information related to colors, their sum, and their occurrences.
+
+**Time Complexity:** $\mathcal{O}(N \log^2 N)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+using ll = long long;
+
+const int MAXN = 1e5 + 1;
+array<ll, MAXN> color, ans;
```

Comment:
separate lines

Reply:
have you never defined arrays on the same line in your life